storage/replica_command.go

```diff
@@ -743,6 +743,19 @@ func (r *Replica) PushTxn(batch engine.Engine, ms *engine.MVCCStats, args proto.
 	if err != nil {
 		return reply, err
 	}
+	// There are three cases in which there is no transaction entry, in decreasing likelihood
+	// * a transaction is still active; it just hasn't committed/restarted/aborted/heartbeat yet
```

Comment:
I'm for "the transaction", although the original author of this comment is @tschottdorf.